01-intro.Rmd

```diff
@@ -124,7 +124,7 @@ Justifiable ethical, legal, and technical factors for limiting sharing of data i
 
 - Informed consent will not permit or will limit the scope or extent of sharing and future research use
 - Existing consent (e.g., for previously collected biospecimens) prohibits sharing or limits the scope or extent of sharing and future research use
-- Privacy or safety of research participants would be compromised or place them at greater risk of re-identification or suffering harm, and protective measures such as de-identification and [Certificates of Confidentiality](https://grants.nih.gov/policy/humansubjects/coc.htm) would be insufficient
+- Privacy or safety of research participants would be compromised or place them at greater risk of re-identification or suffering harm, and protective measures such as de-identification and [Certificates of Confidentiality](https://grants.nih.gov/policy/humansubjects/coc.htm) would be insufficient. _At Fred Hutch, you should get in contact with the [Institutional Review Office](https://centernet.fredhutch.org/cn/u/iro.html){target="_blank"} if you have questions about compliance of your research._
```

Comment:
@carriewright11 how's this? @vortexing could use your Hutch wisdom here :)

Reply:
good - I just also want to scare people away from doing de-identification without expert help :P

Reply:
agreed! Probably best to get _something_ down now, then run it by Hutch experienced folks for a gut check.